frame/benchmark/src/benchmarking.rs

```diff
@@ -0,0 +1,465 @@
+			if i <= n {
+				keys.push(num);
+			}
+			MyHashMap::insert(num, i);
+		}
+
+		// Return the `contains_key_hash_map` n times call
+		Ok((crate::Call::<T>::contains_key_hash_map(keys), RawOrigin::Signed(account::<T>(n))))
+	}
+}
+
+struct ContainsKeyHashMap2;
+impl<T: Trait> BenchmarkingSetup<T, crate::Call<T>, RawOrigin<T::AccountId>> for ContainsKeyHashMap2 {
+
+	fn components(&self) -> Vec<(BenchmarkParameter, u32, u32)> {
+		vec![
+			// Number of reads
+			(BenchmarkParameter::N, 1, 1000),
+		]
+	}
+
+	fn instance(&self, components: &[(BenchmarkParameter, u32)]) -> Result<(crate::Call<T>, RawOrigin<T::AccountId>), &'static str>
+	{
+		// Get N
+		let n = components.iter().find(|&c| c.0 == BenchmarkParameter::N).unwrap().1;
+
+		let mut keys: Vec<u128> = Vec::new();
+
+		let max_size = <ContainsKeyHashMap as BenchmarkingSetup<T, _, _>>::components(&ContainsKeyHashMap)[0].2;
+
+		sp_std::if_std! {
+			println!("{:?}", max_size);
+		}
+
+		for i in 0..n {
+			let hash = sp_io::hashing::twox_128(&i.encode()).into();
+			let num = u128::from_le_bytes(hash);
+			if i <= n {
+				keys.push(num);
+			}
+			MyHashMap::insert(num, i);
+		}
+
+		// Return the `contains_key_hash_map` n times call
+		Ok((crate::Call::<T>::contains_key_hash_map(keys), RawOrigin::Signed(account::<T>(n))))
+	}
+}
+
+struct RemovePrefix;
+impl<T: Trait> BenchmarkingSetup<T, crate::Call<T>, RawOrigin<T::AccountId>> for RemovePrefix {
+
+	fn components(&self) -> Vec<(BenchmarkParameter, u32, u32)> {
+		vec![
+			// Number of reads
+			(BenchmarkParameter::N, 1, 1000),
+		]
+	}
+
+	fn instance(&self, components: &[(BenchmarkParameter, u32)]) -> Result<(crate::Call<T>, RawOrigin<T::AccountId>), &'static str>
+	{
+		// Get N
+		let n = components.iter().find(|&c| c.0 == BenchmarkParameter::N).unwrap().1;
+
+		// Add values to a double map.
+		for i in 0..n {
+			for j in 0..n {
+				MyDoubleMap::insert(i, j, n);
+			}
+		}
+		
+		// Return the `remove_prefix` n times call
+		Ok((crate::Call::<T>::remove_prefix(n), RawOrigin::Signed(account::<T>(n))))
+	}
+}
+
+struct DoNothing;
+impl<T: Trait> BenchmarkingSetup<T, crate::Call<T>, RawOrigin<T::AccountId>> for DoNothing {
+
+	fn components(&self) -> Vec<(BenchmarkParameter, u32, u32)> {
+		vec![
+			// Random input
+			(BenchmarkParameter::N, 1, 1000),
+		]
+	}
+
+	fn instance(&self, components: &[(BenchmarkParameter, u32)]) -> Result<(crate::Call<T>, RawOrigin<T::AccountId>), &'static str>
+	{
+		// Get N
+		let n = components.iter().find(|&c| c.0 == BenchmarkParameter::N).unwrap().1;
+
+		// Return `do_nothing`
+		Ok((crate::Call::<T>::do_nothing(n), RawOrigin::Signed(account::<T>(n))))
+	}
+}
+
+selected_benchmark! {
+	DoNothing,
+	ReadValue,
+	PutValue,
+	ExistsValue,
+	RemoveValue,
+	ReadMap,
+	InsertMap,
+	ContainsKeyMap,
+	ContainsKeyHashMap,
+	ContainsKeyHashMap2,
+	RemovePrefix,
+	AddMemberList,
+	AppendMemberList
+}
+
+impl<T: Trait> Benchmarking<BenchmarkResults> for Module<T> {
```

Comment:
Probably you don't need this function with the new version of `selected_benchmark!`